packages/engine/src/initializeEngine.ts

```diff
@@ -327,7 +317,8 @@ export const initializeEngine = async (initOptions: InitializeOptions = {}): Pro
   } else if (options.type === EngineSystemPresets.EDITOR) {
     Engine.userId = 'editor' as UserId
     Engine.isEditor = true
-    // Engine timer should not start here for eidtor since we load scene after selecting.
+    // Engine timer should not start for eidtor since we load scene after selecting.
+    Engine.engineTimer.stop()
```

Comment:
the engine should be always running. we will need it for other things. loading and unloading scenes should only be changes in state and add/removing entities/components

Reply:
Fixed!